README.md

```diff
@@ -95,6 +95,7 @@ The following lines can be used as swappy's default:
 ## Installation
 
 - [Arch Linux](https://aur.archlinux.org/packages/swappy-git)
+- [fedora 31/32](https://copr.fedorainfracloud.org/coprs/wef/swappy)
```

Comment:
I would prefer `Fedora` here instead of `fedora`, I see the capital Fedora is used on their [website](https://getfedora.org/)

Reply:
OK, done.